server/cli/dataPopulation/Dnd5eDataSeeder.ts

```diff
@@ -0,0 +1,101 @@
+import { v4 as uuidv4 } from 'uuid';
+
+import {
+  Dnd5eApiAdapter,
+  Dnd5eApiClient,
+  Dnd5dApiNormalizer
+} from '../dataPopulation/dnd5eapi';
+import {
+  IThirdPartyDndAdapter,
+  Monster,
+  SourceKinds
+} from './types';
+
+import { MonsterTemplate } from './templates';
+
+import { Creature } from '../../sequelize/db';
+
+// This should likely be moved up a level. There is a need to start sharing types between client and server
+import {
+  MarkdownEntity
+} from '../../../src/core/types';
+
+export class Dnd5eDataSeeder {
+  private adapter: IThirdPartyDndAdapter;
+  private source: SourceKinds;
+
+  constructor(source: SourceKinds) {
+    this.source = source;
+    this.init();
+  }
+
+  async populateMonsters(): Promise<void> {
+    console.log('Starting: populateDndMonsters. This may take a while');
```

Comment:
5e API doesn't have bulk fetching. You need to pull down a list of names and make individual calls to each resource. This is a lengthy process that can take several seconds to complete.